locals.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#--------#
+# Locals #
+#--------#
+locals {
+  random_hash             = "${random_id.random_project_id_suffix.hex}"
+  root_resource_id        = "${var.org_id != "" ? "organizations/${var.org_id}" : var.folder_id != "" ? "folders/${var.folder_id}" : ""}"
+  vpc_host_project_id     = "${var.vpc_host_project_id != "" ? var.vpc_host_project_id : var.project_id}"
+  server_zone             = "${var.server_region}-c"
+  server_startup_script   = "${file("${path.module}/scripts/forseti-server/forseti_server_startup_script.sh")}"
```

Comment:
This path (and the following paths) should have a `.tpl` suffix to indicate that it's templated by Terraform, since Terraform interpolation syntax (e.g. `${}`) is the same as Bash's parameter expansion syntax.